index.html

```diff
@@ -278,47 +418,665 @@ <h4>Identification and Profile Signaling</h4>
+        </p>
+        <p>
+          The namespaces defined by this proposal document are mutable [[namespaceState]]; all undefined names in these namespaces are reserved for future standardization by the W3C.
+        </p>
+      </section>
+      <section id="related-video-object-section">
+        <h3>Related Video Object</h3>
+        <p>
+          A <a>Document Instance</a> MAY be associated with a <a>Related Video Object</a>.
+        </p>
+        <p class="note">
+          While this specification contains specific provisions when a <a>Document Instance</a> is associated with a <a>Related Video Object</a>, it does not prevent the use of a Document Instance with other kinds of Related Media Object, e.g. an audio only object.
+        </p>
       </section>
+      <section id="synchronization-section">
+        <h3>Synchronization</h3>
+        <p class="ednote">Check if we need this, and if we do, rework for audio - e.g. relate to audio sample, or other quantisation units?</p>
+        <p>
+          Each intermediate synchronic document of the Document Instance is intended to be rendered starting on a specific frame and removed by a specific frame of the Related Video Object.
+        </p>
+        <p class="note">In the context of this specification rendering could be visual presentation of text,
+          for example to show an actor what words to speak, or could be audible playback of an audio resource,
+          or could be physical or haptic, such as a Braille display.
+        </p>
+        <p>
+          When mapping a media time expression M to a frame F of a Related Video Object (or Related Media Object), e.g. for the purpose of mixing audio sources signalled by a Document Instance into the main program audio of the <a>Related Video Object</a>, the presentation processor MUST map M to the frame F with the presentation time that is the closest to, but not less, than M.
+        </p>
+        <p>
+          EXAMPLE 1
+          A media time expression of 00:00:05.1 corresponds to frame ceiling( 5.1 × ( 1000 / 1001 × 30) ) = 153 of a <a>Related Video Object</a> with a frame rate of 1000 / 1001 × 30 ≈ 29.97.
+        </p>
+        <p class="note">
+          In typical scenario, the same video program (the <a>Related Video Object</a>) will be used for Document Instance authoring, delivery and user playback.
+          The mapping from media time expression to <a>Related Video Object</a> above allows the author to precisely associate <a>audio description</a> content with video frames, e.g. around existing audio dialogue and sound effects.
+          In circumstances where the video program is downsampled during delivery, the application can specify that, at playback, the relative video object be considered the delivered video program upsampled to is original rate, thereby allowing audio content to be presented at the same temporal locations it was authored.
+        </p>
+      </section>
```

Comment:
Can we remove the entire section?

Reply:
I suggest leaving it and opening an issue to fix it, because it's clearly broken. Or delete it and open an issue to define synchronisation requirements. Happy either way.

Reply:
I opened https://github.com/w3c/dapt/issues/45. I would prefer deleting the section.